go/cmd/vtctldclient/command/transactions.go

```diff
@@ -64,6 +64,21 @@ var (
 
 		DisableFlagsInUseLine: true,
 	}
+
+	readTransactionStateOptions = struct {
+		Dtid string
+	}{}
+
+	// ReadTransactionState makes a ReadTransactionState gRPC call to a vtctld.
+	ReadTransactionState = &cobra.Command{
+		Use:     "read-transaction-state --dtid <dtid>",
+		Short:   "Reads the state of the unresolved transaction by querying each participating shard.",
```

Comment:
nit: can simply call it `read-transaction` or `transaction-info` or `get-info`

Reply:
Done.